documentation/feature/medical-system.md

```diff
@@ -125,83 +144,101 @@ Advanced medical system brings more different types of wounds, each with its own
   * Effects: pain - light, bleeding - slowly.
   
 We also distinguish different types of fractures (WIP/not implemented yet):
+
 * Broken femur
   * Description
   * Sources: bullets, vehicle crashes, backblast, explosions, artillery shells, grenades.
   * Effects: pain - extremely high, unable to stand until healed by medic.
  
 #### 3.1.5 Bandage types
+
 Advanced medical system brings 4 different types of bandages and also properly working tourniquet. Below there is a list of all bandage types with effectiveness vs different wound types. Higher effectiveness bandages needs to be applied fewer times than ones with lower effectiveness. That depends on wound size. Applying bad type of bandage on given wound can yield wound opening after a while if not stiched fast enough.
 All bandage types weights about 50 grams each.
 
-* Bandage (basic)
+* Bandage (basic) is best for:
   * Abrasions - highest effectiveness
-  * Avulsions - lowest effectiveness
-  * Contusions - highest effectiveness
-  * Crush wounds - low effectiveness
-  * Cut wounds - very low effectiveness
   * Lacerations - medium effectiveness
-  * Velocity wounds - lowest effectiveness
-  * Puncture wounds - low effectiveness
 
-* Bandage (packing)
+* Bandage (packing) is best for
   * Abrasions - highest effectiveness
   * Avulsions - highest effectiveness
-  * Contusions - highest effectiveness
-  * Crush wounds - low effectiveness
-  * Cut wounds - lowest effectiveness
-  * Lacerations - lowest effectiveness
   * Velocity wounds - highest effectiveness
-  * Puncture wounds - lowest effectiveness
 
-* Bandage (elastic)
+* Bandage (elastic) is best for:
   * Abrasions - highest effectiveness
-  * Avulsions - lowest effectiveness
-  * Contusions - highest effectiveness
   * Crush wounds - highest effectiveness
   * Cut wounds - highest effectiveness
   * Lacerations - highest effectiveness
-  * Velocity wounds - low effectiveness
   * Puncture wounds - high effectiveness
 
-* QuikClot
-  * Abrasions - medium effectiveness
-  * Avulsions - high effectiveness
-  * Contusions - medium effectiveness
-  * Crush wounds - medium effectiveness
-  * Cut wounds - medium effectiveness
-  * Lacerations - medium effectiveness
-  * Velocity wounds - high effectiveness
-  * Puncture wounds - high effectiveness
+* QuikClot is best for: 
+  * All types of wounds - medium effectiveness
+  * Special: It has lowest wound opening chance. It is often needed to apply it more than once, depening on wound type and size.
 
 * Tourniquet
   * Can only be applied on limbs
   * Stops bleeding from wounds
+  * Should be used only when there are more than one wounds
```

Comment:
I don't think this is necessary. Sure it's the smart thing to do, provided you have the bandages, but there is no actual requirement for this.